contracts/src/internal/HyperdriveLP.sol

```diff
@@ -564,4 +567,42 @@ abstract contract HyperdriveLP is
             _marketState.bondReserves = updatedBondReserves.toUint128();
         }
     }
+
+    function _updateLiquiditySafe(
+        int256 _shareReservesDelta
+    ) internal returns (bool) {
+        // Calculate the updated reserves.
+        uint256 shareReserves_ = _marketState.shareReserves;
+        int256 shareAdjustment_ = _marketState.shareAdjustment;
+        uint256 bondReserves_ = _marketState.bondReserves;
+        (
+            uint256 updatedShareReserves,
+            int256 updatedShareAdjustment,
+            uint256 updatedBondReserves,
+            bool success
+        ) = LPMath.calculateUpdateLiquiditySafe(
+                shareReserves_,
+                shareAdjustment_,
+                bondReserves_,
+                _minimumShareReserves,
+                _shareReservesDelta
+            );
+
+        if (!success) {
+            return false;
+        }
```

Comment:
I know this sounds nit-picky, but the style we use in this codebase is to avoid having "naked" code like this. In this case, I'd suggest removing the newline, so that it is part of the above code block. Otherwise, we'll need to add a comment explaining what this is doing.